test-kit/components/selection-list-driver.ts

```diff
@@ -0,0 +1,36 @@
+import {DriverBase, simulate} from 'test-drive-react';
+import {SelectionList} from '../../src';
+import listBaseStyle from '../../src/components/selection-list/selection-list.st.css';
+import {elementHasStylableClassName, elementHasStylableState} from '../../test/utils/inspect-stylable';
+
+export class SelectionListTestDriver extends DriverBase {
+    public static ComponentClass = SelectionList;
+
+    public focus() {
+        simulate.focus(this.root);
+    }
+
+    public blur() {
+        simulate.blur(this.root);
+    }
+
+    public get items() {
+        return Array.from(this.root.children);
+    }
+
+    public keyDown(keyCode: number) {
+        simulate.keyDown(this.root, {keyCode});
+    }
+
+    public click(element: Element) {
```

Comment:
Yes, `clickItem(idx: number)` was my initial thought too, but then I wanted to add [a test](https://github.com/wix/stylable-components/pull/312/files#diff-77bdd5df824f24fa13ddb390398779a0R92) that clicks on something inside of the item to make sure that we take the `data-value` attribute from the item container, and not from `event.target`. And so I thought it would be better to have a single click function that can click on anything than to create a separate function for each thing that needs to be clicked.